Api/User.py

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
You should have a docstring comment at the beginning of every module with a brief description of what the module implements.